hack/verify-readme.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eu
+
+rootdir="$(echo $(cd $(dirname $0) && pwd))/.."
+
+cd "$rootdir"
+before=$(cat README.md)
+make update-readme
+after=$(cat README.md)
+
+if [[ "$before" != "$after" ]]; then
```

Comment:
I don't want to actually update README.md with this script.

```suggestion
tempfile="$(mktemp)"
cat README.md >"$tempfile"
make update-readme README_FILE="$tempfile" >/dev/null
diff="$(diff -u ./README.md "$tempfile" ||:)"

if [[ -n "$diff" ]]; then
  echo "$diff" >&2
```

Reply:
> I don't want to actually update README.md with this script.

I agree with you!
fixed and rebased 👍 4f321af